doc/sphinx-guides/source/developers/big-data-support.rst

```diff
@@ -120,6 +120,23 @@ To configure the options mentioned above, an administrator must set two JVM opti
 ``./asadmin create-jvm-options "-Ddataverse.files.<id>.public=true"``
 ``./asadmin create-jvm-options "-Ddataverse.files.<id>.ingestsizelimit=<size in bytes>"``
 
+Globus File Transfer
+--------------------
+
+When configured to use a Globus accessible S3 store and interact with a community-developed Dataverse Globus app, Dataverse can support the transfer of files via Globus for upload and download. Due to differences in the access control models of Dataverse and Globus, enabling this capability on a store will disable restriction and embargo capabilities in that store.
```

Comment:
```suggestion
When configured to use a Globus accessible S3 store and interact with a community-developed Dataverse Globus app, Dataverse can support the transfer of files via Globus for upload and download. Due to differences in the access control models of Dataverse and Globus, enabling this capability on a store will disable the ability to restrict and embargo files in that store.
```

I'm suggesting the above because we don't use the term "restriction" much.